Flippybot

karismatic-megafauna opened the pull request:
Not sure if this is going to work, at this point it definitely wont as a lot of this is psuedo code.

Big issues:
- [ ] getting the users.list isn't going to be easy, as this thing just looks at a mesage. Might have to hard code that :( or how do i query the slack api for the gutiboyz? Can hardcode around this, but it will suck :(
- [ ] not sure where to get the message timestamp, how do i see what is actually in the req? Postman?

karismatic-megafauna committed:
Add flippy bot form barely bot, add to app.js

karismatic-megafauna committed:
Add psuedo code for flippy bot

awseward commented on src/app.js:
```diff
@@ -11,6 +12,7 @@ const port = process.env.PORT || 3000;
 app.use(bodyParser.urlencoded({ extended: true }));
 
 app.post("/doesheknower", barelyBot);
+app.post("/doesheknower", flippyBot);
```
I know this is a WIP, but let's do a separate endpoint for this.

michaelghinrichs replied:
good call, but isn't that going to require another slack integration? We are already at our limit :(

awseward replied:
Nope, you can have it hit multiple endpoints.

awseward committed:
Change flippy to flipper

Also use flipper endpoint instead of piggybacking off of doesheknower

awseward committed:
Add function to read SLACK_API_TOKEN env var

awseward committed:
Add request to package.json

awseward committed:
Fetching users, etc

We did a bunch, probably should have committed earlier.
* Fetching users
* Checking if message sender is the flipper (kind of). Committing this
so we can deploy to staging and grab a message and see what's good.

awseward committed:
Only do anything for the flipper

awseward committed:
[REBASE_ME] Checking user fetching

awseward committed:
[REBASE_ME] Prefer leading underscores for non-exported fns

awseward committed:
Merge branch 'master' of github.com:awseward/gutibot into flippybot

Conflicts:
	src/app.js

awseward committed:
Update flipperBot to use gutiBot

awseward committed:
Merge branch 'master' of github.com:awseward/gutibot into flippybot

awseward committed:
Add a TODO list
